### PR TITLE
chore: update tests and change timestamp type

### PR DESCRIPTION
**Pull request by christyjacob4**
- Refactor tests to be more consistent across runs
- Change return type of the timestamp to be consistent across adapters

### CODE REVIEW

File: src/Abuse/Adapters/TimeLimit/Database.php [hunk 1]
```diff
@@ -246,17 +249,18 @@ public function getLogs(?int $offset = null, ?int $limit = 25): array
...
-    public function cleanup(string $datetime): bool
+    public function cleanup(int $timestamp): bool
     {
-        Authorization::skip(function () use ($datetime) {
+        $timestamp = $this->toDateTime($timestamp);
+        Authorization::skip(function () use ($timestamp) {
             do {
                 $documents = $this->db->find(self::COLLECTION, [
-                    Query::lessThan('time', $datetime),
+                    Query::lessThan('time', $timestamp),
```
Comment (Meldiron) [inline src/Abuse/Adapters/TimeLimit/Database.php:263]: Dont we need to change attribute type too?

Comment (christyjacob4) [reply src/Abuse/Adapters/TimeLimit/Database.php:263]: in the database adapter, I [convert the timestamps](https://github.com/utopia-php/abuse/pull/91/files#diff-5251a91329b67cf39cdc8d4eea5d4edeb4b326287ff6df7ed167492cd8239feaR139) to the required format

File: tests/Abuse/Base.php [hunk 2]
```diff
@@ -4,67 +4,116 @@
...
-        $value = '0.0.0.10';
-
-        $adapter = $this->getAdapter($key, 1, 1);
-        $adapter->setParam($key, $value);
-        $this->abuseIp = new Abuse($adapter);
-        $this->assertEquals($this->abuseIp->check(), false);
-        $this->assertEquals($this->abuseIp->check(), true);
-
-        sleep(1);
+        $adapter = $this->getAdapter('dyamic-key-{{ip}}', 2, 1);
```
Comment (loks0n) [inline tests/Abuse/Base.php:30]: Should this be `dynamic-key-...

**Review by loks0n — APPROVED** [review]

File: tests/Abuse/Base.php (hunk 2 above)
Comment (christyjacob4) [reply tests/Abuse/Base.php:30]: Updated 👍

**Pull request merged**